apps/omg_watcher_rpc/lib/web/controllers/fee.ex

```diff
@@ -0,0 +1,33 @@
+# Copyright 2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.WatcherRPC.Web.Controller.Fee do
+  @moduledoc """
+  Operations related to fees.
+  """
+
+  use OMG.WatcherRPC.Web, :controller
+
+  alias OMG.Watcher.HttpRPC.Client
+
+  def get_all(conn, params) do
+    with {:ok, _} <- expect(params, "currencies", list: &to_currency/1),
+         child_chain_url <- Application.get_env(:omg_watcher, :child_chain_url),
+         {:ok, fees} <- Client.get_fees(params, child_chain_url) do
+      api_response([skip_hex_encode: fees], conn, :all_fees)
```

Comment:
`[skip_hex_encode: fees]` :point_left:  this is nice - but again should be in the `view` by convention.
Thanks!